src/components/segmentedControl/index.tsx

```diff
@@ -157,11 +165,13 @@ const SegmentedControl = (props: SegmentedControlProps) => {
 
   const animatedStyle = useAnimatedStyle(() => {
     if (segmentsStyle.value.length !== 0) {
-      const isFirstElementSelected = animatedSelectedIndex.value === 0;
-      const isLastElementSelected = animatedSelectedIndex.value === segmentsStyle.value.length - 1;
-      const xOffset = isFirstElementSelected ? -2 : isLastElementSelected ? 2 : 0;
-      const inset = withTiming(segmentsStyle.value[animatedSelectedIndex.value].x + xOffset, TIMING_CONFIG);
-      const width = withTiming(segmentsStyle.value[animatedSelectedIndex.value].width * BORDER_WIDTH, TIMING_CONFIG);
+      const isFirst = animatedSelectedIndex.value === 0;
```

Comment:
Why the naming change? 'isFirst' is not the same as 'isFirstSelected'...

Reply:
Changed.